### PR TITLE
feat: email the snapshot to input email w/signoff

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR fixes #

- [ ] Yes, I signed my commits.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -154,3 +157,41 @@ runs:
...
+          - **Review the Snapshot**: Ensure all components are correctly represented and the infrastructure layout aligns with your expectations.
+          - **Share with Your Team**: Use this snapshot to facilitate discussions and planning with your team.
+          - **Explore Further**: Learn more about how MeshMap can help visualize and manage your infrastructure by visiting our [documentation](https://docs.layer5.io/meshmap).
+
+          If you have any questions or need further assistance, please feel free to reach out to our support team.
+
+          Thank you for using MeshMap!
+
+          Best regards,  
+          The MeshMap Team
```
Comment (leecalcote) [inline action.yml:193]: No such team.

**Comment by leecalcote** [comment]
Please stop using "successfully".

File: action.yml (hunk 1 above)
Comment (Jougan-0) [reply action.yml:193]: The Layer5 Team?

**Comment by Jougan-0** [comment]
> Please stop using "successfully".

A successfully again slipped through even I paid attention while writing cloud errors and logs still one had to slip through. :rage:

**Comment by leecalcote** [comment]
Markdown isn't going to display correctly in email...

File: action.yml [hunk 2]
```diff
@@ -154,3 +157,41 @@ runs:
...
+
+          ---
+          Meshery by Layer5 - The Cloud Native Management Plane.
+          Learn more at [https://layer5.io/meshery](https://layer5.io/meshery)
```
Comment (leecalcote) [inline action.yml]:
```suggestion
        subject: "Your Meshery Snapshot"
        from: '"Layer5" <${{ secrets.MAIL_USERNAME }}>'  
        to: ${{ inputs.email }}  
        body: |
          <p>Hello,</p>

         <p>Your ${{ inputs.application_type }} has been converted to a Meshery Design. Your snapshot has been generated and is now available.</p>
          
          <p><a href="${{ env.RESOURCE_URL }}"><img src="${{ env.RESOURCE_URL }}" width="80%" /></a></p>
          <p><b>Next Steps</b></p>
          <ul>
  <ol><b>Edit your snapshot</b>: Create or sign into your <a href="https://meshery.layer5.io">Layer5 Cloud</a> account to edit your snapshot.</ol>
  <ol><b>Share with your team</b>: Use this snapshot to facilitate discussions and planning with your team.</ol>
  <ol><b>Add Snapshots to your GitHub Pull Requests</b>: <a href="http://meshery.layer5.io/connect/github/new">Connect Meshery Snapshots with your GitHub repo</a> and see changes pull request-to-pull request. Get snapshots of your infrastructure directly in your PRs.</ol>
</ul>

          <p>Thank you for using Meshery! If you enjoy this tool, share with your friends.</p>

          ---<br />
          Meshery by Layer5 - The Collaborative Cloud Native Manager.  Learn more at <a href="https://layer5.io/meshery">layer5.io</a>.
```

File: action.yml [hunk 3]
```diff
@@ -154,3 +157,41 @@ runs:
...
```
Comment (leecalcote) [inline action.yml:197]:
```suggestion
        subject: "Your Meshery Snapshot"
        from: '"Layer5" <${{ secrets.MAIL_USERNAME }}>'  
        to: ${{ inputs.email }}  
        body: |
          <p>Hello,</p>

         <p>Your ${{ inputs.application_type }} has been converted to a Meshery Design. Your snapshot has been generated and is now available.</p>
          
          <p><a href="${{ env.RESOURCE_URL }}"><img src="${{ env.RESOURCE_URL }}" width="80%" /></a></p>
          <p><b>Next Steps</b></p>
          <ul>
  <ol><b>Edit your snapshot</b>: Create or sign into your <a href="https://meshery.layer5.io">Layer5 Cloud</a> account to edit your snapshot.</ol>
  <ol><b>Share with your team</b>: Use this snapshot to facilitate discussions and planning with your team.</ol>
  <ol><b>Add Snapshots to your GitHub Pull Requests</b>: <a href="http://meshery.layer5.io/connect/github/new">Connect Meshery Snapshots with your GitHub repo</a> and see changes pull request-to-pull request. Get snapshots of your infrastructure directly in your PRs.</ol>
</ul>

          <p>Thank you for using Meshery! If you enjoy this tool, share with your friends.</p>

          ---<br />
          Meshery by Layer5 - The Collaborative Cloud Native Manager.  Learn more at <a href="https://layer5.io/meshery">layer5.io</a>.
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**